src/js/components/Meter/doc.js

```diff
@@ -26,10 +15,21 @@ export const doc = Meter => {
 
   DocumentedMeter.propTypes = {
     ...genericProps,
-    background: backgroundPropType.defaultValue({
-      color: 'light-2',
-      opacity: 'medium',
-    }),
+    background: PropTypes.oneOfType([
+      PropTypes.string,
+      PropTypes.shape({
+        color: PropTypes.string,
+        opacity: PropTypes.oneOfType([
+          PropTypes.oneOf(['weak', 'medium', 'strong']),
+          PropTypes.bool,
+        ]),
```

Comment:
I think opacity should also accept a number or a string, e.g. `opacity=0.8`

Reply:
Gotcha, I'll check into how Meter works -- these are just the values that already existed and I was moving them into the Meter file. Will address this in a separate PR.